quantized_mesh_encoder/encode.py

```diff
@@ -17,7 +21,8 @@ def encode(
         indices,
         bounds=None,
```

Comment:
Note to self, I'd love to add `*` above `bounds` so that `positions` and `indices` are the only positional arguments.